apps/src/schoolInfoManager.js

```diff
@@ -0,0 +1,381 @@
+import $ from 'jquery';
+
+export default function SchoolInfoManager(existingOptions) {
+
+  var districtListFirstLoad = true;
+
+  var districtElement = $('#school-district-id');
+
+  function scrollToBottom() {
+    if (!existingOptions['suppressScrolling']) {
```

Comment:
A little bit surprised our linter doesnt complain about this, vs. `existingOptions.suppressScrolling` but I guess this was existing code and it didnt complain about it then :man_shrugging: